js/src/components/notices.js

```diff
@@ -12,11 +12,7 @@ const Notices = {
 					dismiss[ 0 ].addEventListener( 'click', ( ev ) => {
 						notice.style.height = notice.offsetHeight + 'px';
 						ev.preventDefault();
-						// Give the event a slight delay to allow the height to
```

Comment:
@dugajean, I'd get @DavidCramer input on this one.
He told me that this was needed to be able to get the height of the notifications before the animation.

Reply:
@pereirinha Yes, I put in the delay/timeout to aid in calculating heights to allow the animation to fire correctly.
With this change, I can confirm the animations don't work anymore. There's also no reason to edit the notice JS in this PR as it's unrelated to the gallery.

Reply:
@dugajean 
Let's rever this change, please.

Reply:
That animation was very clunky and bad to look at. That's the reason I removed it.

Reply:
Plus the solution to use many `setTimeout` invocations is not very clean. This can be achieved with CSS animations if really necessary.

Reply:
The question here has more to do with the scope of the PR.
I don't think that one should address multiple subjects. If this were a separated PR, it would help the discussion and alternatives approach.